pkg/v1/providers/infrastructure-vsphere/v1.1.0/ytt/base-template.yaml

```diff
@@ -149,11 +151,11 @@ spec:
             - name: vip_interface
               value: '${ VIP_NETWORK_INTERFACE }'
             - name: vip_leaseduration
-              value: "30"
+              value: "15"
             - name: vip_renewdeadline
-              value: "20"
+              value: "10"
             - name: vip_retryperiod
-              value: "4"
```

Comment:
This needs to be reverted back. We need to relax the leader election parameters.